### PR TITLE
Fix/metrics 0.12

### PR DESCRIPTION
**Pull request by lukanus**

### CODE REVIEW

File: api/metrics.go [hunk 1]
```diff
@@ -4,40 +4,53 @@ import "github.com/figment-networks/indexing-engine/metrics"
...
 
-	rawRequestDuration = metrics.MustNewHistogramWithTags(metrics.HistogramOptions{
-		Namespace: "indexers",
-		Subsystem: "worker_api_kava",
-		Name:      "request_duration",
+	rawRequestHTTPDuration = metrics.MustNewHistogramWithTags(metrics.HistogramOptions{
+		Namespace: "indexerworker",
+		Subsystem: "api",
+		Name:      "request_http",
+		Desc:      "Duration how long it takes to take data from cosmos",
```
Comment (pacmessica) [inline api/metrics.go:18]: from cosmos?

File: api/metrics.go [hunk 2]
```diff
@@ -4,40 +4,53 @@ import "github.com/figment-networks/indexing-engine/metrics"
...
 		Tags:      []string{"endpoint", "status"},
 	})
 
 	numberOfItems = metrics.MustNewHistogramWithTags(metrics.HistogramOptions{
-		Namespace: "indexers",
-		Subsystem: "worker_api_kava",
-		Name:      "number_of_items",
+		Namespace: "indexerworker",
+		Subsystem: "api",
+		Name:      "tx_num",
```
Comment (pacmessica) [inline api/metrics.go:33]: these seem like reusable fields/metrics. When we want to add metrics to cosmos-worker et al, where should we move this?

File: api/transaction.go [hunk 3]
```diff
@@ -382,7 +386,13 @@ func rawToTransaction(ctx context.Context, c *Client, in []types.TxResponse, blo
 			}
 
 			if err != nil {
-				c.logger.Error("[COSMOS-API] Problem decoding transaction ", zap.Error(err), zap.String("type", msg.Type()), zap.String("route", msg.Route()))
+				if errors.Is(err, errUnknownMessageType) {
```
Comment (pacmessica) [inline api/transaction.go:389]: 👍

**Review by pacmessica — APPROVED** [review]

File: api/metrics.go (hunk 1 above)
Comment (lukanus) [reply api/metrics.go:18]: From the outerspace! :P

File: api/metrics.go (hunk 2 above)
Comment (lukanus) [reply api/metrics.go:33]: Hard to say really, let's copy paste is for now, We may need to investigate this in future, does it make sense to move this to separate repo.

**Pull request merged**